reify-transformer.js

```diff
@@ -0,0 +1,2 @@
+require('reify');
```

Comment:
If you can check with the right order, it should work (`--require async-to-gen/register --require reify`)